### ISSUE TITLE
Mirroring 1999.io HTML file to host in an S3 bucket or my own server?

### ISSUE THREAD

**Issue by yaeloss** [scripting/1999-project#40]
My 1999 server is running great. I've created several posts, and linked to them in my [Freedom Controller](http://oss.yael.at/microblog.html) app and on my Twitter and Facebook.

The problem I get is that when I'm away from my computer, and I'm not actively logged into the terminal with storage.js running, [my posts](yaeloss.com:1999/users/YaelOss/) don't appear.

I've run the command as DW suggested (sudo npm install forever -g), but I get the following error:

> npm WARN optional Skipping failed optional dependency /forever/chokidar/fsevents:
> npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.2.9

I understand that's not too big of a deal if I'm just able to replicate the [HTML file](http://yaeloss.com:1999/users/YaelOss/index.html) that my 1999 server produces, but I just can't figure out how to mirror that file from my instance server to an S3 bucket or my personal server.

I know @scripting provided [details on mapping to a blog](https://github.com/scripting/1999-project/blob/master/docs/domains.md), but I can't get it working without using the port numbe in the domain.

I set up the instance with Digital Ocean. I tried another with EC2, but got too many errors.

I've seen that there was [some documentation](https://twitter.com/YaelOss/status/1180976766866857985) on this, but it seems Andy Sylvester's node isn't running all the time either ;)

[Ron Chester in his very exhaustive overview of 1999 resources](http://reader.smallpicture.com/?opmlurl=https%3A%2F%2Fdl.dropbox.com%2Fs%2Fjybig60wiguxmfq%2F1999ioBloggingArchive.opml%3Fdl%3D0) linked to this post, but it doesn't work: http://node.andysylvester.com:1999/users/AndySylvester99/2016/06/26/0036.html

I've tried to use "rclone" to mirror the file, but to no avail as it's just too complicated.

I feel that this issue is easier to solve than my current path, so I appreciate the help.

TL;DR How do I copy the HTML file that my 1999.io server outputs to an s3 bucket or via FTP to my own sever?

**Comment by yaeloss** [comment]
Added to that, the output for sharing my 1999 blog online with a port number "yaeloss.com:1999/users/Yaeloss" just isn't very attractive – hence why I want to mirror to a separate domain!

**Comment by scripting** [comment]
1999.io is a dynamic server, so there is no HTML file to mirror to a static server. 

You cover a lot of points, but if you want help we have to focus. So start here, **you absolutely must get forever to run on your server**. Without that nothing is going to work properly.

So let's start at the beginning. Where are you hosting your server? It sounds like Digital Ocean, but that's not 100 percent clear. 

If it is Digital Ocean, I promise you, you can get forever running there. I have lots of servers there and they all run forever. 

Also there's no problem getting your site accessible over port 80. My 1999 server works that way, http://friends.farm/.

**Comment by yaeloss** [comment]
I use Digital Ocean. After I upgraded the ubuntu instance, I was able to get the "forever" running.

Current Output: 

> /usr/local/bin/forever -> /usr/local/lib/node_modules/forever/bin/forever
> /usr/local/lib
> └── forever@1.0.0

**Comment by yaeloss** [comment]
Now that I have that, I guess my next question is on mapping the domain.

https://github.com/scripting/1999-project/blob/master/docs/domains.md

I've changed in the config.json. Now, for the DNS of my subdomain, what is the value for the CNAME?
